src/js/relayclient/RelayClient.js

```diff
@@ -135,20 +129,17 @@ class RelayClient {
     const relayRequestAbiEncode = relayHub.methods.relayCall(relayRequestOrig, sig, approvalData).encodeABI()
 
     if (
-      utils.isSameAddress(returnedTx.to, relayHubAddress) &&
-      relayRequestAbiEncode === returnedTx.input &&
+      utils.isSameAddress(ethUtils.bufferToHex(tx.to), relayHubAddress) &&
+      relayRequestAbiEncode === ethUtils.bufferToHex(tx.data) &&
```

Comment:
Not sure what you mean, but I changed the return value of the server to be the raw Tx, which is easy to decode by ` const tx = new Transaction(returnedTx)` in line 106.

Reply:
I mean you still have "returnedTx" available, and it was "returnedTx.to" before you changed the line. 
Is there a reason to go through "ethUtils.bufferToHex(tx.to)" instead of "returnedTx.to"?

Reply:
returnedTx is now raw hex.